pom.xml

```diff
@@ -40,8 +40,8 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
     <version.org.awaitility-awaitility>3.0.0</version.org.awaitility-awaitility>
-    <version.io.opentracing>0.22.0</version.io.opentracing>
-    <version.io.opentracing.contrib-opentracing-web-servlet-filter>0.0.8</version.io.opentracing.contrib-opentracing-web-servlet-filter>
+    <version.io.opentracing>0.30.0</version.io.opentracing>
+    <version.io.opentracing.contrib-opentracing-web-servlet-filter>0.0.9-SNAPSHOT</version.io.opentracing.contrib-opentracing-web-servlet-filter>
```

Comment:
Currently using snapshot version of servlet-filter which also includes use of OT java 0.30.0.